Selenium/YandexLogin/Pages/BasePage.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+
+namespace YandexLogin.Pages
+{
+    public class BasePage
+    {
+        private IWebDriver driver;
+        private WebDriverWait wait;
+
+        public BasePage(IWebDriver driver)
+        {
+            this.driver = driver;
+            this.wait = new WebDriverWait(this.driver, TimeSpan.FromSeconds(10));
+            this.driver.Manage().Window.Maximize();
+        }
+
+        public IWebDriver Driver
```

Comment:
Instead of this construction use [auto-property](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties)

Reply:
Fixed